core/controllers/profile.py

```diff
@@ -604,6 +605,12 @@ def get(self):
                 'user_is_logged_in': False
             })
 
+    @acl_decorators.open_access
+    def post(self):
```

Comment:
POST request should have some payload variable indicating that the request was made to update set_user_has_viewed_lesson_info_once and not some other variable

Reply:
Done!